block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf

```diff
@@ -46,7 +46,7 @@ GOPATH=$GOPATH  					# Read from environment
 # Helm chart install
```

Comment:
The vars such as `PVG_BX_DASH_S`, `PVG_CLUSTER_CRUISER`, `BLOCK_PLUGIN_HOME` and 
```
IMAGE_REGISTRY=registry.stage1.ng.bluemix.net
USER_NAMESPACE=contsto2
PLUGIN_IMAGE=ibmcloud-block-storage-attacher
PLUGIN_BUILD=$PLUGIN_BUILD
```
are not updated in this jp-tok.conf file

Reply:
In this file those lines were commented . Un commented and updated